source/docs/appendices/migration-guides.md

```diff
@@ -0,0 +1,42 @@
+---
+id: migration-guides
+title: Migration Guides
+---
+
+This area will contain the Migration steps to follow for upgrading your store to new Front-Commerce versions.
+
+Our goal is to make migrations as smooth as possible. This is why we try to make many changes backword compatible by using deprecation warnings. The deprecation warnings are usually removed in the next breaking release.
+
+## `1.0.0-alpha.1` -> `1.0.0-alpha.2`
+
+### Atoms refactoring ([#178](https://gitlab.com/front-commerce/front-commerce/issues/178))
+
+We are in the process of rewriting our CSS classes to make it easier for new external contributors to dive into Front-Commerce. However this is a lot of work because of the many features already implemented in Front-Commerce. Thus, we've splitted this in 5 smaller iterations (see ([#97](https://gitlab.com/front-commerce/front-commerce/issues/97)) for more details).
```

Comment:
```suggestion
One of the goal of `1.0.0` is to rewrite our CSS classes to make it easier for new external contributors to dive into Front-Commerce. However this is a lot of work because of the many features already implemented in Front-Commerce. Thus, we've splitted this in 5 smaller iterations (see ([#97](https://gitlab.com/front-commerce/front-commerce/issues/97)) for more details). This release is the first step toward this goal.
```